api/controllers/tasks.py

```diff
@@ -136,6 +140,20 @@ def process_proposal(credentials, tpid):
         tpm.dbs.commit()
         logger.info("Added round (%s)" % (r.id))
 
+    else:
```

Comment:
Do you think we should also show rejected tasks to the user who proposed it along with the changes requested in some UI (maybe in the `Your Proposals` table in the `Tasks`), instead of just sending an email?  

(this could be done in a separate PR, not sure what others think of this though)

Reply:
That is a pretty good idea. But having those tasks might increase database usage by quite a bit, and if the user is going to make changes in a new proposal, having the old rejected proposal might not serve much of a purpose.